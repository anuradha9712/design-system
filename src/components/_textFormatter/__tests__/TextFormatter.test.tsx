import * as React from 'react';
import { render } from '@testing-library/react';
import { Bold, Italics, Underline } from '@/index';

describe('Bold component', () => {
  it('should match the snapshot', () => {
    const { baseElement } = render(<Bold>Bold Text</Bold>);
    expect(baseElement).toMatchSnapshot();
  });
});

describe('Italics component', () => {
  it('should match the snapshot', () => {
    const { baseElement } = render(<Italics>Italics text</Italics>);
    expect(baseElement).toMatchSnapshot();
  });
});

describe('Underline component', () => {
  it('should match the snapshot', () => {
    const { baseElement } = render(<Underline>Underline Text</Underline>);
    expect(baseElement).toMatchSnapshot();
  });
});
