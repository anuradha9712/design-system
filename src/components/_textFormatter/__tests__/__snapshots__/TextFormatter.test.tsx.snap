// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bold component should match the snapshot 1`] = `
<body>
  <div>
    <b
      data-test="DesignSystem-Bold"
    >
      Bold Text
    </b>
  </div>
</body>
`;

exports[`Italics component should match the snapshot 1`] = `
<body>
  <div>
    <i
      data-test="DesignSystem-Italics"
    >
      Italics text
    </i>
  </div>
</body>
`;

exports[`Underline component should match the snapshot 1`] = `
<body>
  <div>
    <u
      data-test="DesignSystem-Underline"
    >
      Underline Text
    </u>
  </div>
</body>
`;
